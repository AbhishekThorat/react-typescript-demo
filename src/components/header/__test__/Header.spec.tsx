import * as React from "react";
import { shallow } from "enzyme";
import 'jest-extended';
import * as renderer from 'react-test-renderer';

import { Header } from "../";

it("Check hard coded heading", () => {
    expect(shallow(<Header />).text()).toEqualCaseInsensitive('BBC RADIO');
});

it('Snapshot testing for header component', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});