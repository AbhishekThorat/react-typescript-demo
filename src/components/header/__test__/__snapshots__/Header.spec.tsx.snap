// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing for header component 1`] = `
<div
  className="sc-bdVaJa esZbpe"
>
  <div
    className="sc-bwzfXH PPgmw"
  >
    BBC RADIO
  </div>
</div>
`;
